pupil-spa/src/app/aa-colours/aa-colours.component.spec.ts

```diff
@@ -8,13 +8,16 @@ import { StorageServiceMock } from '../services/storage/storage.service.mock';
 import { RouteService } from '../services/route/route.service';
 import { RouteServiceMock } from '../services/route/route.service.mock';
 import { PupilPrefsService } from '../services/pupil-prefs/pupil-prefs.service';
+import { PupilPrefsSyncService } from '../services/pupil-prefs-sync/pupil-prefs-sync.service';
```

Comment:
`PupilPrefsService` is a service responsible for submitting selected pupil prefs to the queue.

`PupilPrefsSyncService` is a service responsible for looking at local storage for prefs that have been set:
- during the same session by the pupil
- during a previous check and appear in the current check's config
or provides default selections in case none of the above exist.

@GuyHarwood Perhaps the first service could be renamed PupilPrefsSubmissionService for clarity.

Reply:
it shouldn't really be called 'sync' as its synonymous with the backend service that is actually doing the sync work and confused me.  Unless it creates a cluttered service that is doing more than one thing i would have thought it would be easier to just have one pupil prefs service in the SPA.  It feels like you have a prefs 'data' service for loading items out of storage, and another one that submits items to the queue.

Reply:
@GuyHarwood @jon-shipley I have merged the two services into a single `PupilPrefsService` and renamed the method from sync -> loadPupilPrefs.